coordinator/src/main/java/org/apache/uniffle/coordinator/access/checker/AbstractAccessChecker.java

```diff
@@ -25,6 +25,11 @@
 public abstract class AbstractAccessChecker implements AccessChecker {
 
   protected AbstractAccessChecker(AccessManager accessManager) throws Exception {
-    
+
+  }
+
+  @Override
+  public void refreshAccessChecker() {
```

Comment:
Could we make it  as an abstract method?

Reply:
> Could we make it as an abstract method?

Changing refreshAccessChecker to an abstract method and requiring all classes that inherit from AbstractAccessChecker to implement, it is not appropriate.

Reply:
ok